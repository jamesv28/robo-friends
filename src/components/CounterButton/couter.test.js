import React from 'react';
import CouterButton from './counter.button';
import {shallow} from 'enzyme';

it('should take look at the snapshot of the counter button', () => {
    const mockColor = 'red';
    expect(shallow(<CouterButton color={mockColor} />)).toMatchSnapshot();
})

it('correctly increments the counter', () => {
    const mockColor = 'red';
    const wrapper = shallow(<CouterButton color={mockColor} />);

    wrapper.find('[id="counter"]').simulate('click');
    expect(wrapper.state()).toEqual({counter: 2});
})