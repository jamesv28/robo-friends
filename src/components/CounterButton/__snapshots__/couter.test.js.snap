// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take look at the snapshot of the counter button 1`] = `ShallowWrapper {}`;
